Sources/Concordium/Domain/Block.swift

```diff
@@ -1,15 +1,13 @@
 import Foundation
 
-public typealias BlockHash = Data // 32 bytes
-
-public enum BlockIdentifier {
+public enum BlockIdentifier: ToGRPC {
```

Comment:
maybe we should update protocol naming. Make this protocol name like `GRPCConvertible`

Reply:
Yeah I had something similar to that at one point, but then I realised not all types need to convert both ways (there is also a `FromGRPC` protocol.

In most cases we only need `FromGRPC` as that is meant for translating types returned from the GRPC API, whereas `ToGRPC` is meant for translating input into the types required by the generated GRPC client.

Reply:
Sure, let's leave it as it is for now. Perhaps later we can come up with a better name for this protocol.